### PR TITLE
add TiFlash spill doc

### PR DESCRIPTION
**Pull request by windtalker**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add three tiflash configurations.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -200,6 +200,15 @@ delta_index_cache_size = 0
...
     dt_page_gc_threshold = 0.5
 
+    ## 从 v7.0.0 引入，表示带 group by key 的 HashAggregation 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashAggregation 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
+    max_bytes_before_external_group_by = 0
+
+    ## 从 v7.0.0 引入，表示 sort/topN 算子在触发 spill 之前的最大可用内存，超过该阈值之后 sort/TopN 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
+    max_bytes_before_external_sort = 0
+
+    ## 从 v7.0.0 引入，表示带等值 join 条件的 HashJoin 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashJoin 算子会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
+    max_bytes_before_external_sort = 0
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:210]: `max_bytes_before_external_sort = 0` is the same as line 207. Please check.

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -200,6 +200,15 @@ delta_index_cache_size = 0
     ## 从 v6.2.0 引入，表示 PageStorage 单个数据文件中有效数据的最低比例。当某个数据文件的有效数据比例低于该值时，会触发 GC 对该文件的数据进行整理。默认为 0.5。
     dt_page_gc_threshold = 0.5
 
+    ## 从 v7.0.0 引入，表示带 group by key 的 HashAggregation 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashAggregation 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
+    max_bytes_before_external_group_by = 0
+
+    ## 从 v7.0.0 引入，表示 sort/topN 算子在触发 spill 之前的最大可用内存，超过该阈值之后 sort/TopN 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:206]:
```suggestion
    ## 从 v7.0.0 引入，表示 sort/topN 算子在触发 spill 之前的最大可用内存，超过该阈值之后 sort/TopN 会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制。
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -200,6 +200,15 @@ delta_index_cache_size = 0
...
+    max_bytes_before_external_sort = 0
+
+    ## 从 v7.0.0 引入，表示带等值 join 条件的 HashJoin 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashJoin 算子会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:209]:
```suggestion
    ## 从 v7.0.0 引入，表示带等值 join 条件的 HashJoin 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashJoin 算子会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制。
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -200,6 +200,15 @@ delta_index_cache_size = 0
     ## 从 v6.2.0 引入，表示 PageStorage 单个数据文件中有效数据的最低比例。当某个数据文件的有效数据比例低于该值时，会触发 GC 对该文件的数据进行整理。默认为 0.5。
     dt_page_gc_threshold = 0.5
 
+    ## 从 v7.0.0 引入，表示带 group by key 的 HashAggregation 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashAggregation 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:203]:
```suggestion
    ## 从 v7.0.0 引入，表示带 group by key 的 HashAggregation 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashAggregation 会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制。
```

File: tiflash/tiflash-configuration.md (hunk 1 above)
Comment (windtalker) [reply tiflash/tiflash-configuration.md:210]: Should be `max_bytes_before_external_join`

**Review by joey-yez — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

**Review by joey-yez — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**